config/serverless.oblt.yml

```diff
@@ -34,6 +34,10 @@ xpack.observability.createO11yGenericFeatureId: true
 ## APM Serverless Onboarding flow
 xpack.apm.serverlessOnboarding: true
 
+# Synthetics mTLS cert locations
+xpack.uptime.service.tls.certificate: /mnt/elastic-internal/http-certs/tls.crt
+xpack.uptime.service.tls.key: /mnt/elastic-internal/http-certs/tls.key
```

Comment:
I'm not sure how to feel about that one. What will be the behavior when running locally an observability project via `--serverless=observability`?. 

Per-project type config files are supposed to be for "generic" (not "env" dependent) configurations. For settings depending on running in MKI, we set them in from the k8s project/kibana controller instead.

Reply:
> What will be the behavior when running locally an observability project via --serverless=observability?

Nothing should be impacted because these values are only needed when running in a cloud Project. For local development, we either hit a locally-running service in Minikube (which does not use mTLS), or we hit the QA service, which supports basic auth, and is configured in `kibana.dev.yml`.

For QA/Staging/Prod, these paths are the same so no need to configure separate paths. We could, of course, include them via kibana controller but as I understand it we'd be injecting the same key/value for each one in all envs.